packages/crypto/src/primitives/secp256k1.ts

```diff
@@ -663,30 +713,28 @@ export class Secp256k1 {
    * not assess whether the key corresponds to a known entity or its security status (e.g., whether
    * it has been compromised).
    *
-   * Example usage:
-   *
+   * @example
    * ```ts
-   * const privateKey = new Uint8Array([...]); // A 32-byte private key
-   * const isValid = await Secp256k1.validatePrivateKey({ key: privateKey });
+   * const privateKeyBytes = new Uint8Array([...]); // A 32-byte private key
+   * const isValid = await Secp256k1.validatePrivateKey({ privateKeyBytes });
    * console.log(isValid); // true or false based on the key's validity
    * ```
    *
-   * @param options - The options for the key validation.
-   * @param options.key - The private key to validate, represented as a Uint8Array.
+   * @param params - The parameters for the key validation.
+   * @param params.privateKeyBytes - The private key to validate, represented as a Uint8Array.
    *
    * @returns A Promise that resolves to a boolean indicating whether the private key is valid.
    */
-  private static async validatePrivateKey(options: {
-    key: Uint8Array
+  private static async validatePrivateKey({ privateKeyBytes }: {
```

Comment:
is this supposed to be public?

Reply:
Good question.  It was initially but was marked private when working on the new `Signer`, `AsymmetricKeyGenerator`, etc. types and attempting to get these classes to "satisfy" those type definition constraints.  While it technically worked, it had the unfortunate side effect of making TSDocs disappear (longer discussion about anonymous classes).  Will mark as public for now and see how much usage these functions get.